frontend/src/pages/ModelTester/ChartComponenet2/ChartComponent2.jsx

```diff
@@ -27,11 +27,49 @@ const ChartComponent2 = forwardRef(({ chartData }, ref) => {
                         ticks: {
                             autoSkip: false, // Ensure all x-axis labels are shown
                         },
+                        title: {
+                            display: true,   // Show the title
+                            text: 'Model Name', // Set the x-axis title
+                            font: {
+                                size: 15,   // Increase the font size
+                                weight: 'bold',
+                            },
+                        }
                     },
                     y: {
                         min: 0,
                         max: 1,
                         beginAtZero: true, // Ensure y-axis starts at zero
+                        title: {
+                            display: true,   // Show the title
```

Comment:
looks good! can we do this for chart component1 as well?

Reply:
Yup, I already did that!

Reply:
great thank you!